src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -73,35 +106,36 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
     return logGroupsArray
 }
 
-export function createLogGroupPrompter(logGroups: string[]) {
-    const options = logGroups.map<DataQuickPickItem<string>>(logGroupString => ({
-        label: logGroupString,
-        data: logGroupString,
-    }))
-
-    return createQuickPick(options, {
-        title: 'Select Log Group',
-        placeholder: 'Enter text here',
-    })
-}
-
 export function createFilterpatternPrompter() {
     return createInputBox({
         title: 'Keyword Search',
         placeholder: 'Enter text here',
     })
 }
 
+export function createRegionSubmenu() {
+    return new RegionSubmenu(getLogGroupsFromRegion)
+}
+
 export interface SearchLogGroupWizardResponse {
-    logGroup: string
+    submenuResponse: RegionSubmenuResponse<string>
     filterPattern: string
 }
 
 export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
-    public constructor(logGroups: string[]) {
-        super()
+    public constructor(logGroupInfo?: CloudWatchLogsGroupInfo) {
+        logGroupInfo
+            ? super({
+                  initState: {
+                      submenuResponse: {
+                          data: logGroupInfo.groupName,
+                          region: logGroupInfo.regionName,
+                      },
+                  },
+              })
+            : super()
```

Comment:
Huh I'm actually kind of surprised Typescript let's you do this. Just shows that the control-flow analysis is pretty good. But anyway this is probably more readable:
```suggestion
        super({
            initState: {
                submenuResponse: logGroupInfo ? {
                    data: logGroupInfo.groupName,
                    region: logGroupInfo.regionName,
                } : undefined,
            },
        })
```